HeterogeneousCore/CUDAUtilities/interface/cudaCheck.h

```diff
@@ -45,6 +46,11 @@ namespace cms {
       return false;
     }
 
+    inline bool cudaCheck_(
+        const char* file, int line, const char* cmd, CUresult result, const std::string& description) {
```

Comment:
In principle the additional overload could be avoided by changing the existing function signatures to use `std::string_view` along 
```cpp
inline bool cudaCheck_(const char* file, int line, const char* cmd, CUresult result, std::string_view description = std::string_view()) {
```
That would increase the minimum required C++ version for this file to 17, which to my understanding should be ok nowadays for `nvcc`.

@fwyzard Do you have a preference?

Reply:
implemented in a separate commit 43043d403b6c065005a3ae110d39a9519282a72d so it can be reverted easily if necessary